diff --git a/scripts/git-version.sh b/scripts/git-version.sh
index a60cdc4..0a7abc6 100755
--- a/scripts/git-version.sh
+++ b/scripts/git-version.sh
@@ -1,21 +1,3 @@
 #!/bin/sh -e
 
-# parse the current git commit hash
-COMMIT=`git rev-parse HEAD`
-
-# check if the current commit has a matching tag
-TAG=$(git describe --exact-match --abbrev=0 --tags ${COMMIT} 2> /dev/null || true)
-
-# use the matching tag as the version, if available
-if [ -z "$TAG" ]; then
-    VERSION=$COMMIT
-else
-    VERSION=$TAG
-fi
-
-# check for changed files (not untracked files)
-if [ -n "$(git diff --shortstat 2> /dev/null | tail -n1)" ]; then
-    VERSION="${VERSION}-dirty"
-fi
-
-echo $VERSION
+echo v0.8.0
